credible/cli-overview.mdx

```diff
@@ -0,0 +1,166 @@
+  assertions/
+    src/              # Assertion source files (.a.sol)
+    test/             # Test files (.t.sol)
+  src/
+    MyProtocol.sol    # Protocol smart contracts
+  test/
+    MyProtocol.t.sol  # Protocol tests
+```
+
+### Create a Project
+
+Use the [Credible Layer dApp](/credible/glossary#credible-dapp) to create a project and link your contracts.
+
+## Authentication & Configuration
+
+### How Authentication Works
+
+`pcl` uses OAuth-based authentication:
+
+1. Run `pcl auth login`
+2. A URL and authentication code are displayed
+3. Visit the URL in your browser
+4. Connect your wallet and approve the authentication
+5. `pcl` automatically detects successful authentication
+
+### Configuration
+
+Configuration is stored in `~/.pcl/config.toml` and includes:
+
+- Authentication token (persists across sessions)
+- Pending assertions for submission (assertions that have been stored but not yet submitted)
+- User address (the address of the wallet that is authenticated)
+
+You can view your configuration with `pcl config show` or delete it with `pcl config delete`.
+
+## Assertion Lifecycle
+
+The complete assertion lifecycle from development to deployment:
+
+### Development & Testing
+
+1. **Write** your assertion contract in Solidity (`.a.sol` files). See the [Assertion Guide](/credible/pcl-assertion-guide) to learn how to write assertions.
+2. **Test** locally using `pcl test` to verify your assertion logic (`.t.sol` files). See the [Testing Assertions guide](/credible/testing-assertions) for details.
+3. **Build** your contracts using `pcl build` (optional, useful for checking compilation errors; `store` will build automatically if needed)
+
+<Note>
+When updating an existing assertion contract, modify your code, test with `pcl test`, then store and submit the updated version and activate in the dApp.
+</Note>
+
+### Authenticate → Store → Submit → Activate
+
+Once your assertion contract is [tested and ready](/credible/testing-assertions):
+
+1. **Authenticate**: Ensure you're authenticated with `pcl auth login` (required before storing or submitting)
+   ```bash
+   pcl auth login
+   ```
+
+2. **Store**: Upload assertion bytecode and source code to Assertion DA
+   ```bash
+   pcl store MyAssertionContract 0xADDRESS_OF_SMART_CONTRACT
+   ```
+
+3. **Submit**: Register the assertion contract with your project in the dApp
+   ```bash
+   pcl submit -a 'MyAssertionContract(0xADDRESS_OF_SMART_CONTRACT)' -p my-project
+   ```
+
+   <Note>
+   To submit multiple assertion contracts, use interactive mode (`pcl submit` without options) to select which stored assertion contracts to submit and which project to submit them to.
+   </Note>
+
+4. **Activate**: Complete the activation process in the dApp by linking the assertion contract to your protocol's contract. This makes the assertion live on-chain and begins protecting your contract. See the [dApp Guide](/credible/dapp-guide#linking-assertions-to-your-project) for details.
```

Comment:
Let's follow a unified glossary -- please talk with Cindy to align on the glossary. We have aligned that this is "deploying an assertion" and then it's going to be "enforced" after a specific "timelock".

We need to ensure that we talk about the same things in the same way across all the docs